lammps/src/USER-COLVARS/fix_colvars.cpp

```diff
@@ -473,6 +473,18 @@ void FixColvars::one_time_init()
 
 /* ---------------------------------------------------------------------- */
 
+int FixColvars::modify_param(int narg, char **arg)
+{
+  if (strcmp(arg[0],"add_config") == 0) {
```

Comment:
I would probably make this just `config` instead of `add_config` for consistency with VMD and NAMD: we should allow initializing the module with an empty config, which is all specified with subsequent commands.  Other than that it looks great.

Reply:
Actually, this should probably be `configfile`, not `config`, which provides an inlined configuration string.

@akohlmey A good template for the `config` option seems to be the `here` option of the `python` command.  Do you have a better suggestion?

Reply:
there is one issue. you cannot do fix_modify without an instance of the proxy class. so i had to add a test for that.

python style "heredocs" are a general feature of the parser. so you can have something like:
```
fix_modify 1 config """
<some multi-line config file content>
...
"""
```
and everything inside the triple double quotes becomes one arg[] string item.
single line would work as well:
```
fix_modify 1 config "<come single config option"
```

Reply:
i'll make some changes to the submitted PR and you can check it out. From there on, it should be easy for you or jerome or both of you to sort out the details and add as many gimmicks as useful.

Reply:
Absolutely. Your template is clear enough, so feel free to add anything you think useful, or leave the rest to us.

I have yet to test the code, and most importantly I need to merge changes in the LAMMPS repository so that they are not undone at the next update.

Reply:
if you synchronize colvars with LAMMPS. please check out warnings issued from `-Wunused-parameters` and comment out the variable name, like this:
```
@@ -645,7 +670,7 @@ void FixColvars::setup(int vflag)
 /* ---------------------------------------------------------------------- */
 /* Main colvars handler:
  * Send coodinates and add colvar forces to atoms. */
-void FixColvars::post_force(int vflag)
+void FixColvars::post_force(int /* vflag */)
 {
   // some housekeeping: update status of the proxy as needed.
   if (me == 0) {
```

We've transformed most of LAMMPS this way already

Reply:
Understood.  I synced the doc text and Install script so far.